check-diff.sh

```diff
@@ -548,7 +547,7 @@ function run_phpunit_local {
 			if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
 				phpunit $( if [ -n "$PHPUNIT_CONFIG" ]; then echo -c "$PHPUNIT_CONFIG"; fi )
 			fi
-			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' -mindepth 2 ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
+			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
```

Comment:
I do remember having issues where the unit tests were being ran twice with Customize Posts, though I don't recall if that is still an issue. I'll checkout this PR and run the pre-commit hook to verify. Just have to get VVV fixed on my desktop. It's currently not able to run unit tests. 😞 

@MattGeri could you update the PR? There are merge conflicts and it would be good to test against the latest codebase with these enhancements added.